src/store/getters.js

```diff
@@ -115,13 +110,17 @@ export default {
       ? TX_DIRECTION.sent
       : TX_DIRECTION.received;
   },
-  getDexContracts: (_, { activeNetwork }) => (DEX_CONTRACTS[activeNetwork.networkId]),
-  getAccountPendingTransactions: (
-    { transactions: { pending } }, { activeNetwork, account: { address } },
-  ) => (pending[activeNetwork.networkId]?.length ? pending[activeNetwork.networkId]
-    ?.filter((transaction) => transaction.tx.callerId === address
-    || transaction.tx.senderId === address
-    || transaction.tx.recipientId === address
-    || transaction.recipientId === address
-    || transaction.recipient === address) : []),
+  getAccountPendingTransactions: (state, getters) => {
+    const { nodeNetworkId } = useSdk({ store: { state, getters } });
+    const { address } = getters.account;
```

Comment:
I'd love to, but using the composables within the store getters is quirky and I'd like to avoid it for now. Take a look how I pass the `store` to the `useSdk` in line above - this is rather a workaround.

Reply:
this is why I get confused. you could use `useSdk` but not `useAccounts`.

Different rules for each composable child😃

Reply:
Topic covered on priv... resolving.